docs/tech-preview.rst

```diff
@@ -1,7 +1,7 @@
 Tech previews
 =============
 
-The following features are currently being released as part of a tech preview:
+The following features are currently being released as a part of a tech preview:
```

Comment:
This was correct before...

Reply:
No. https://www.oxfordlearnersdictionaries.com/definition/english/part_1?q=part